uniffi_bindgen/src/scaffolding.rs

```diff
@@ -22,6 +22,18 @@ mod filters {
     use super::*;
     use std::fmt;
 
+    pub fn choose(
+        expr_value: &bool,
+        then_value: &dyn fmt::Display,
+        else_value: &dyn fmt::Display,
+    ) -> Result<String, askama::Error> {
+        Ok(if *expr_value {
+            then_value.to_string()
+        } else {
+            else_value.to_string()
+        })
+    }
```

Comment:
Note to self: raise an issue with @djc to add a default filter to askama.

Reply:
For simple cases, you might just be able to use `then_value.unwrap_or(else_value.to_string())` in the template, but I think it could some facilities to Askama to facilitate patterns like this.

Reply:
> `then_value.unwrap_or(else_value.to_string())`

I don't _think_ this is what I'm asking for. Your suggestion in trinary form looks like: 

`then_value != null ? then_value : else_value`

I'm looking for 

`expr_value ? then_value : else_value`

You _can_ express what my implementation does in askama:

```
{% let string %}
{% if expr_value %}
{% let string = "then value" %}
{% else %}
{% let string = "else value" %}
```

But it's quite verbose.